core/load_balancer_rule_test.go

```diff
@@ -13,6 +13,8 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+func ptr[T any](x T) *T { return &x }
```

Comment:
Nitpick: We've already got a `boolPtr` helper method available and used widely, so might as well use that instead of introducing this. Alternatively we could swap all the `<something>Ptr` helpers to just use this `ptr` helper, but that might be overkill and a bit too far off topic for this PR.